__tests__/endpoints.test.js

```diff
@@ -301,7 +426,7 @@ describe("/api/articles/:article_id/comments", () => {
           article_id: 1,
           author: expect.any(String),
           votes: expect.any(Number),
-          created_at: expect.any(String),
+          created_at: expect.any(String), // shouldn't this be a number?
```

Comment:
Don't leave unnecessary comments in code, but created_at is indeed a date string.